doc/leaderf.txt

```diff
@@ -624,6 +624,12 @@ g:Lf_EmptyQuery                                   *g:Lf_EmptyQuery*
 
     Default value is 1.
 
+g:Lf_IgnoreBufferName                             *g:Lf_IgnoreBufferName*
+    This option specifies whether to remove the current buffer name from
+    the result list.
+
+    Default value is 1.
```

Comment:
I don't this feature is enabled by default, I think it's better that the default value is 0.